Makefile

```diff
@@ -19,7 +19,10 @@ lint:
 	pre-commit run -a --hook-stage manual
 
 test:
-	pytest tests
+	@./test-coverage.sh
+
+test-coverage:
+	@./test-coverage.sh
```

Comment:
are these just duplicates?

Reply:
Well I started off with a new make `rule` and then when working on the CircleCI config I realised that to make sure all CircleCI tasks pick up the coverage shaabang, I decided to change that line too. 

What would you prefer, `test` rule that also does coverage or a new rule that's called `test-coverage`? If I'm right, currently we don't have one.